src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -113,6 +115,32 @@ function isPreview() {
     return $(document).find('.navbar').text().indexOf('Preview') !== -1;
 }
 
+function isModeration() {
+    return $('#moderationHintButton').length !== 0;
+}
+
+function bindModerationHintButton() {
+    if (!isModeration()) {
+        return;
+    }
+    
+    var expandText = '[Show more]';
+    var closeText = '[Show less]';
+    $('#moderationHintButton').text(expandText);
```

Comment:
Rather than adding the default text by javascript, my preference would be that the default text is written directly in the `<a>` tag in the `feedbackSubmissionEdit.tag` file. This is so that devs know the default text simply by looking at the `.tag` file rather than having to reference the `.js` file.